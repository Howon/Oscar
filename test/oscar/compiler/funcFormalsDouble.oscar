def foo(d: double) => unit = {
	println(d);
}

def main() => unit = {
	foo(42.0);
}
