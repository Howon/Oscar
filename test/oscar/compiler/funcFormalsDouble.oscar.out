42.0000000000
